src/layout/grid/gridColumn.jsx

```diff
@@ -1,119 +1,391 @@
+    switch (breakpoint) {
+        /* eslint-disable no-param-reassign */
+        case 'mobile':
+            globalStyles[theme.breakpoints.down(374)] = styles;
+
+            break;
+        case 'mobile-medium':
+            globalStyles[theme.breakpoints.up(375)] = styles;
+
+            break;
+        case 'mobile-large':
+            globalStyles[theme.breakpoints.up(425)] = styles;
+
+            break;
+        case 'laptop':
+            globalStyles[theme.breakpoints.up(1024)] = styles;
+
+            break;
+        case 'desktop':
+            globalStyles[theme.breakpoints.up(1440)] = styles;
+
+            break;
+        case 'desktop-large':
+            globalStyles[theme.breakpoints.up(1720)] = styles;
+
+            break;
+        case 'sm':
+            Object.assign(globalStyles, styles);
+
+            break;
+        default:
+            globalStyles[theme.breakpoints.up(breakpoint)] = styles;
+        /* eslint-enable no-param-reassign */
+    }
+};
+
+const useStyles = makeStyles((theme) => ({
+    root: {
+        margin: 0,
+        /**
+         * Deprecated classses
+         */
+        '&$deprecatedAlignStretch': {
+            alignSelf: 'stretch',
+        },
+        '&deprecatedFloatedLeft': {
+            marginRight: 'auto',
+        },
+        '&$deprecatedFloatedRight': {
+            marginLeft: 'auto',
+        },
+        '&$deprecatedTextAlignCenter': {
+            textAlign: 'center',
+        },
+        '&$deprecatedTextAlignLeft': {
+            textAlign: 'left',
+        },
+        '&$deprecatedTextAlignRight': {
+            textAlign: 'right',
+        },
+        '&$deprecatedVerticalAlignBottom': {
+            alignItems: 'bottom',
+            display: 'flex',
+        },
+        '&$deprecatedVerticalAlignCenter': {
+            alignItems: 'center',
+            display: 'flex',
+        },
+        '&$deprecatedVerticalAlignTop': {
+            alignItems: 'top',
+            display: 'flex',
+        },
+        /**
+         * End of deprecated classes
+         */
+    },
+    /**
+     * Deprecated classses
+     */
+    deprecatedAlignStretch: {},
+    deprecatedFloatedLeft: {},
+    deprecatedFloatedRight: {},
+    deprecatedTextAlignCenter: {},
+    deprecatedTextAlignLeft: {},
+    deprecatedTextAlignRight: {},
+    deprecatedVerticalAlignBottom: {},
+    deprecatedVerticalAlignCenter: {},
+    deprecatedVerticalAlignTop: {},
+    /**
+     * End of deprecated classes
+     */
+    ...reduce([
+        ...theme.breakpoints.keys,
+        'mobile',
+        'mobile-medium',
+        'mobile-large',
+        'laptop',
+        'desktop',
+        'desktop-large',
+    ], (accumulator, key) => {
+        generateGrid(accumulator, theme, key);
+
+        return accumulator;
+    }, {}),
+}));
+
+const GridColumn = React.forwardRef(
+    /* eslint-disable react-hooks/rules-of-hooks */
+    // eslint-disable-next-line prefer-arrow-callback
+    function gridColumn(props, ref) {
```

Comment:
Should this be capitalized? `GridColumn`

Reply:
Yes.

Reply:
Fixed.

Reply:
Looks good.  Resolving.